examples/mosaic-integration/src/vizzes/Earthquakes.ts

```diff
@@ -0,0 +1,50 @@
+import * as vg from '@uwdata/vgplot';
+import * as topojson from 'topojson-client';
+import { Viz } from '../Viz';
+
+// Requires the MotherDuck sample_data database to be present.
+// It is automatically shared to all new MotherDuck accounts, but if you've detached it you can get it back with the following:
+// ATTACH 'md:_share/sample_data/23b0d623-1361-421d-ae77-62d701d471e6';
+// More details: https://motherduck.com/docs/sample-data-queries/attach-sample-database/
+
+export class EarthquakeViz implements Viz {
+  private land: any | null = null;
+  async initialize() {
+    // Materialize the data for the viz into a local temp table.
+    await vg.coordinator().exec(
+      "CREATE TEMP TABLE IF NOT EXISTS earthquakes_viz AS SELECT * FROM mosaic_examples.main.earthquakes"
+    );
+    const land = await fetch("https://raw.githubusercontent.com/uwdata/mosaic/1faee8bcd386f341df937dc161a0fbc67db29596/docs/public/data/countries-110m.json")
```

Comment:
A thought: we could add this JSON file to the "public" folder (under mosaic-integration) and just fetch it using a relative URL (`fetch ("/countries-110m.json")`). That would shield us in case this changes, and avoid fetching stuff from other sites.